janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/formats/cassandra/CqlBridgeRecordReader.java

```diff
@@ -0,0 +1,377 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
@mbrukman Since this file is a modified version of Apache Cassandra's [CqlRecordReader](https://github.com/apache/cassandra/blob/8b3a60b9a7dbefeecc06bace617279612ec7092d/src/java/org/apache/cassandra/hadoop/cql3/CqlRecordReader.java) should the original (e.g. ASF) copyright header be preserved here? Their header references a [NOTICE](https://github.com/apache/cassandra/blob/8b3a60b9a7dbefeecc06bace617279612ec7092d/NOTICE.txt) file that lists a number of copyright holders (though ASF may be the only applicable one).